sympy/geometry/hyperbola.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sympy
+import random
+import math
+import decimal
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Neither numpy not matplotlib are dependencies of SymPy, so this must be handled with differently.

Reply:
Are all these imports even used?